src/components/shared/DetailsWidget/detailsWidgetConfig.tsx

```diff
@@ -237,19 +236,12 @@ const getDetailItems = (
 ): DetailItemConfig[] => {
   const detailItemsMap = getDetailItemsMap(colony, actionData);
   const detailItemKeys = getDetailItemsKeys(actionData.type);
-  const motionDetailItem = getMotionDetailItem(
-    colony,
-    actionData.fromDomain?.nativeId,
-  );
-  const detailItems = detailItemKeys
-    .map((itemKey) => detailItemsMap[itemKey])
-    .filter((detail) => !!detail.item);
 
-  if (actionData.isMotion) {
-    detailItems.splice(1, 0, motionDetailItem);
-  }
+  const detailItems = detailItemKeys
+    .map<DetailItemConfig | undefined>((itemKey) => detailItemsMap[itemKey])
+    .filter((detail) => !!detail?.item);
 
-  return detailItems;
+  return detailItems as DetailItemConfig[];
```

Comment:
Could the casting be moved to a generic accepted by `filter`? 

```js
.filter<DetailItemConfig>()
```

Reply:
@jakubcolony 

This doesn't work:

``` ts
.filter<DetailItemConfig>((detail) => !!detail?.item)
```

Because the type paramter is telling the filter function that the return value of the predicate function will be `DetailItemConfig`, when in fact it's a boolean. 

Type guards, however, are compatible with filtering, so the following achieves the intended type inference:

``` ts
.filter((detail): detail is DetailItemConfig => !!detail?.item);
```

Reply:
My bad, ignore me then